lib/messaging/messagehandler_message.rb

```diff
@@ -10,4 +10,9 @@ def initialize
     handled = false
   end
 
+  def handle_message(message)
+    message.handled = true
+    puts 'This is probably where the message gets handled.'
```

Comment:
This is as far as I was comfortable going, as I don't know how the GCode / serial stuff works. Input appreciated.